### ISSUE TITLE
Dependency doesn't have a label

### ISSUE THREAD

**Issue by reckart** [lapps/vocabulary-pages#17]
Shouldn't Dependency have a property called "label" like Constituent?

**Comment by nancyide** [comment]
Yes, we use it but it was left out of the vocab page. I am editing now and we will regenerate the pages later today.
